Bottom: 93618208aa33153ce4b7eef1f9f4ad32668523e5
Top:    d44e51cd8cc4c3f5fd9eee26cc3699874980573c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-01-20 18:12:58 +0000

Refresh of butcher-the-xor-crosshair-draw

---

diff --git a/src/crosshair.c b/src/crosshair.c
index 22c1c0d..b012ffb 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -380,7 +380,7 @@ DrawAttached (void)
             {
               gui->set_color (Crosshair.GC, Settings.CrossColor);
 
-              draw_line = Crisshair.AttachedLine;
+              draw_line = Crosshair.AttachedLine;
               draw_line.Thickness = PCB->RatDraw ? 10 : Settings.LineThickness + 2 * (PCB->Bloat + 1);
               thindraw_pcb_line (draw_line);
